WooCommerce/Classes/Authentication/AuthenticationConstants.swift

```diff
@@ -35,4 +35,18 @@ struct AuthenticationConstants {
         "Log in with your WordPress.com account to manage your WooCommerce stores.",
         comment: "Sign in instructions for logging in with a username and password."
     )
+
+    /// Title of "Continue With WordPress.com" button in Login Prologue
+    //
+    static let continueWithWPButtontitle = NSLocalizedString(
```

Comment:
nit:

```suggestion
    static let continueWithWPButtonTitle = NSLocalizedString(
```

Reply:
Wow, thanks! I have been staring at the suggestion for a while and couldn't see the capitalization error. Will push an update for it